nexus/library/qmcpack_converters.py

```diff
@@ -686,16 +826,42 @@ class Convert4qmc(Simulation):
     application_properties = set(['serial'])
     application_results    = set(['orbitals','particles'])
 
+
     def set_app_name(self,app_name):
         self.app_name = app_name
         self.input.set_app_name(app_name)
     #end def set_app_name
 
+
     def propagate_identifier(self):
         None
         #self.input.prefix = self.identifier
     #end def propagate_identifier
 
+
+    def get_prefix(self):
+        input = self.input
+        prefix = 'sample'
+        if input.prefix is not None:
+            prefix = input.prefix
+        #end if
+        return prefix
+    #end def get_prefix
+
+
+    def list_output_files(self):
+        # try to support both pre and post v3.3.0 convert4qmc
+        prefix = self.get_prefix()
+        wfn_file  = prefix+'.Gaussian-G2.xml'
+        ptcl_file = prefix+'.Gaussian-G2.ptcl.xml'
+        if not os.path.exists(os.path.join(self.locdir,ptcl_file)):
+            wfn_file  = prefix+'.wfnoj.xml'
```

Comment:
The new converter produces both .wfnoj.xml and .wfj.xml files by default.  If only one is selected by Nexus, I would think the .wfj.xml is more useful as part of a pipeline.

Reply:
Nexus generally writes its own Jastrow at user request.  The workflow dependency that is being processed by the code in question involves only the orbitals (i.e. qmc.depends(gms,'orbitals')) so the wavefunction without Jastrow is what is desired.